[tf-psa-crypto] Define MBEDTLS_PK_ALG_ECDSA (2/2) (#388)
Mbed-TLS/TF-PSA-Crypto: ChangeLog.d/issue345.txt

Features
   * Helper symbol MBEDTLS_PK_ALG_ECDSA is added to the public "mbedtls/pk.h"

>      header file. It allows to automatically select the proper ECDSA algorithm,

Reviewer: Tiny grammar point - missing something between 'allows' and 'to', e.g.
```suggestion
     header file. It allows the automatic selection of the proper ECDSA algorithm,
```

Features
   * Helper symbol MBEDTLS_PK_ALG_ECDSA is added to the public "mbedtls/pk.h"
     header file. It allows the automatic selection of the proper ECDSA
     algorithm, deterministic or randomized, based on

>      #MBEDTLS_ECDSA_DETERMINISTIC.

Reviewer 2: Not quite, see my comment on the definition.
